xtask/src/cargo.rs

```diff
@@ -19,6 +19,13 @@ pub fn run(args: &[String], cwd: &Path) -> Result<()> {
         bail!("The `cwd` argument MUST be a directory");
     }
 
+    // Make sure to not use a UNC as CWD!
+    // That would make `OUT_DIR` a UNC which will trigger things like the one fixed in https://github.com/dtolnay/rustversion/pull/51
+    // While it's fixed in `rustversion` it's not fixed for other crates we are
+    // using now or in future!
+    #[cfg(target_os = "windows")]
```

Comment:
We can safely use `target_os` since the `xtask` won't be cross-compiled. Otherwise, we would need to infer the HOST_OS